website/docs/r/http_source.html.markdown

```diff
@@ -24,6 +24,14 @@ resource "sumologic_http_source" "http_source" {
   }
 }
 
+resource "sumologic_http_traces_source" "http_traces_source" {
```

Comment:
This should be `sumologic_http_source` resource itself.

Reply:
Good catch up dated the wrong property

Reply:
Fixed